book/src/csi-noide-info-object.md

```diff
@@ -0,0 +1,59 @@
+# CSI CRD: `CSINodeInfo` Object
+
+## What is the `CSINodeInfo` object?
+
+CSI drivers generate node specific information. Instead of storing this in the Kubernetes `Node` API Object, a new CSI specfici Kubenretes CRD was create, the `CSINodeInfo` CRD.
```

Comment:
typo: specific

Reply:
typo: Kubernetes

Reply:
Fixed.

Reply:
I still see `specfici` and `Kubenretes`. Perhaps the changes aren't pushed yet?